### DISCUSSION TITLE
boot stops

### DISCUSSION THREAD

**Discussion by Kerapp** [elementary/triage#476] [Q&A]
![IMG_1278](https://github.com/elementary/triage/assets/162161046/5a777ba9-6a13-4078-ba3a-2ee247cb1321)
Leaving work on Friday I downloaded the latest updates and shut down, as I usually do. Starting up this morning the boot stopped and gave me a BusyBox prompt - (initramfs) _.  I ran advanced recovery mode and got "File system check of the root filesystem failed      The root filesystem on/dev....-root requires a manual fsck"

How do I do that?

**Comment by jfarid27** [comment]
I'm also having a similar issue so bumping.

I don't know much about the ubuntu update paths, but something totally broke my installation. I just get dropped into initramfs and I'm not sure what to do. I have hit "exit", but for me "/dev/mapper/data-root does not exist".

Does anyone have any write ups on how to mount drives and start the OS from initramfs for Elementary? I'm just not sure what to do in this state, but I'm pretty sure it's an easy fix. I did download a linux bootrepair disk and look at the drives. The data is there, it's just something is now messed up during mounting the drive.
